src/content/en/2019/page-weight.md

```diff
@@ -6,809 +6,765 @@ description: Page Weight chapter of the 2019 Web Almanac covering why page weigh
+
+<figure>
+  <table>
+    <tr>
+      <th>Percentile</th>
+      <th>GIF</th>
+      <th>ICO</th>
+      <th>JPG</th>
+      <th>PNG</th>
+      <th>SVG</th>
+      <th>WEBP</th>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>0.27</td>
+      <td>0.31</td>
+      <td>3.08</td>
+      <td>0.4</td>
+      <td>0.28</td>
+      <td>2.1</td>
+    </tr>
+    <tr>
+      <td>25</td>
+      <td>0.75</td>
+      <td>0.6</td>
+      <td>7.7</td>
+      <td>1.17</td>
+      <td>0.46</td>
+      <td>4.4</td>
+    </tr>
+    <tr>
+      <td>50</td>
+      <td>2.14</td>
+      <td>1.12</td>
+      <td>20.47</td>
+      <td>4.35</td>
+      <td>0.95</td>
+      <td>11.54</td>
+    </tr>
+    <tr>
+      <td>75</td>
+      <td>7.34</td>
+      <td>4.19</td>
+      <td>61.13</td>
+      <td>21.39</td>
+      <td>2.67</td>
+      <td>31.21</td>
+    </tr>
+    <tr>
+      <td>90</td>
+      <td>35</td>
+      <td>14.73</td>
+      <td>155.46</td>
+      <td>91.02</td>
+      <td>8.26</td>
+      <td>76.43</td>
+    </tr>
+  </table>
+  <figcaption>Figure 11. File size by image format for images > 100 bytes</figcaption>
+</figure>
+
+#### File size by image format for images > 1024 bytes
+
+<figure>
+  <table>
+    <tr>
+      <th>Percentile</th>
+      <th>GIF</th>
+      <th>ICO</th>
+      <th>JPG</th>
+      <th>PNG</th>
+      <th>SVG</th>
+      <th>WEBP</th>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>1.28</td>
+      <td>1.12</td>
+      <td>3.4</td>
+      <td>1.5</td>
+      <td>1.2</td>
+      <td>3.08</td>
+    </tr>
+    <tr>
+      <td>25</td>
+      <td>1.9</td>
+      <td>1.12</td>
+      <td>8.21</td>
+      <td>2.88</td>
+      <td>1.52</td>
+      <td>5</td>
+    </tr>
+    <tr>
+      <td>50</td>
+      <td>4.01</td>
+      <td>2.49</td>
+      <td>21.19</td>
+      <td>8.33</td>
+      <td>2.81</td>
+      <td>12.52</td>
+    </tr>
+    <tr>
+      <td>75</td>
+      <td>11.92</td>
+      <td>7.87</td>
+      <td>62.54</td>
+      <td>33.17</td>
+      <td>6.88</td>
+      <td>32.83</td>
+    </tr>
+    <tr>
+      <td>90</td>
+      <td>67.15</td>
+      <td>22.13</td>
+      <td>157.96</td>
+      <td>127.15</td>
+      <td>19.06</td>
+      <td>79.53</td>
+    </tr>
+  </table>
+  <figcaption>Figure 12. File size by image format for images > 1024 bytes</figcaption>
+</figure>
 
 The low file size of PNG images compared to JPEG images may seem surprising. JPEG uses [lossy compression](https://en.wikipedia.org/wiki/Lossy_compression) (lossy compression results in data loss; this makes it possible to achieve smaller file sizes) while PNG uses [lossless compression](https://en.wikipedia.org/wiki/Lossless_compression) (lossless compression does not result in data loss; this produces higher-quality, but larger images). However this difference in file sizes is probably a reflection of the popularity of PNGs for iconography due to their transparency support; rather than differences in their encoding and compression.
```

Comment:
```suggestion
The low file size of PNG images compared to JPEG images may seem surprising. JPEG uses [lossy compression](https://en.wikipedia.org/wiki/Lossy_compression). Lossy compression results in data loss, which makes it possible to achieve smaller file sizes. Meanwhile, PNG uses [lossless compression](https://en.wikipedia.org/wiki/Lossless_compression). This does not result in data loss, which this produces higher-quality, but larger images. However, this difference in file sizes is probably a reflection of the popularity of PNGs for iconography due to their transparency support, rather than differences in their encoding and compression.
```